app/client/lib/ethereum/observeBlocks.js

```diff
@@ -109,4 +110,5 @@ observeLatestBlocks = function(){
     peerCountIntervalId = setInterval(function() {
         getPeerCount()
     }, 1000);
+    console.timeEnd('observeLatestBlocks');    
```

Comment:
Don't understand why this section is being timed since some of the code is asynchronously executed anyway. Surely the timing code should go into `updateBalances()` instead?